src/bcd/CMakeLists.txt

```diff
@@ -61,6 +61,15 @@ add_library (bcd STATIC ${sources})
 apply_static_lib_linker_flags (bcd)
 
 
+#--------------------------------------------------------------------------------------------------
+# Static libraries.
+#--------------------------------------------------------------------------------------------------
+
+link_against_platform (bcd)
+
```

Comment:
You can remove the blank line here.

Reply:
Are you sure about this? The spacing I used in this commit matches the style of this file and other files where I see these macros used.

Reply:
Your code is perfectly formatted, very much appreciated. I was just suggesting grouping the two linking commands but it's such a minute detail, forget about it :)